pandas/core/config_init.py

```diff
@@ -249,6 +249,27 @@
     method. Valid values: False,True
 """
 
+pc_latex_multicolumn = """
+: bool
+    This specifies if the to_latex method of a Dataframe uses multicolumns
+    to pretty-print MultiIndex columns.
+    method. Valid values: False,True
```

Comment:
Ah, yes it is. Copied it from longtable or escape which are both still ill-formatted in the current master. I'll fix them too while i'm at it.